napari_cellseg3d/launch_review.py

```diff
@@ -263,29 +267,23 @@ def update_canvas_canvas(viewer, event):
                 print(e)
 
     # Qt widget defined in docker.py
-    dmg = Datamanager(parent=view1)
+    dmg = Datamanager(parent=viewer)
     dmg.prepare(r_path, filetype, model_type, checkbox, as_folder)
-    view1.window.add_dock_widget(dmg, name=" ", area="left")
+    viewer.window.add_dock_widget(dmg, name=" ", area="left")
 
     def update_button(axis_event):
 
         slice_num = axis_event.value[0]
         print(f"slice num is {slice_num}")
         dmg.update(slice_num)
 
-    view1.dims.events.current_step.connect(update_button)
-
-    def crop_volume_around_point(points, layer):
+    viewer.dims.events.current_step.connect(update_button)
 
+    def crop_volume_around_point(points, layer, zoom_factor):
         if zoom_factor != [1, 1, 1]:
             vol = np.array(layer.data, dtype=np.int16)
```

Comment:
I think the datatypes of all arrays dealt with here should be changed from int16 to uint16

Reply:
I get this with uint16 :
can't convert np.ndarray of type numpy.uint16. The only supported types are: float64, float32, float16, complex64, complex128, int64, int32, int16, int8, uint8, and bool.

Reply:
Most likely due to the way utils.resize works (Uses MONAI, ndimage would be cleaner but MONAI transform is actually faster)